spec/models/conversation_spec.rb

```diff
@@ -110,64 +101,51 @@
       end
 
       it 'creates a message' do
-        expect{
-          Diaspora::Parser.from_xml(@xml).receive(@user1, @user2.person)
+        expect {
+          Diaspora::Parser.from_xml(xml).receive(user1, user2.person)
         }.to change(Message, :count).by(1)
       end
       it 'creates a conversation' do
-        expect{
-          Diaspora::Parser.from_xml(@xml).receive(@user1, @user2.person)
+        expect {
+          Diaspora::Parser.from_xml(xml).receive(user1, user2.person)
         }.to change(Conversation, :count).by(1)
       end
       it 'creates appropriate visibilities' do
-        expect{
-          Diaspora::Parser.from_xml(@xml).receive(@user1, @user2.person)
-        }.to change(ConversationVisibility, :count).by(@participant_ids.size)
+        expect {
+          Diaspora::Parser.from_xml(xml).receive(user1, user2.person)
+        }.to change(ConversationVisibility, :count).by(participant_ids.size)
       end
       it 'does not save before receive' do
-        expect(Diaspora::Parser.from_xml(@xml).persisted?).to be false
+        expect(Diaspora::Parser.from_xml(xml).persisted?).to be false
       end
       it 'notifies for the message' do
         expect(Notification).to receive(:notify).once
-        Diaspora::Parser.from_xml(@xml).receive(@user1, @user2.person)
+        Diaspora::Parser.from_xml(xml).receive(user1, user2.person)
       end
     end
   end
 
   describe "#invalid parameters" do
     context "local author" do
-      before do
-        @invalid_hash = {
-          author:              peter.person,
-          participant_ids:     [peter.person.id, @user1.person.id],
-          subject:             "cool stuff",
-          messages_attributes: [{author: peter.person, text: "hey"}]
-        }
-      end
+      let(:invalid_hash) {{author: peter.person, participant_ids: [peter.person.id, user1.person.id],
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.